pom.xml

```diff
@@ -24,6 +36,17 @@
             <artifactId>spring-jdbc</artifactId>
             <version>${spring.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-beans</artifactId>
+            <version>${spring.version}</version>
+        </dependency>
+        <dependency>
```

Comment:
Won't the `spring-beans` and the `spring-context` dependencies be added transitively? Do we really have to declare them explicitly?